From 5978abd6e37f0c4157814ba1fd543a5aed879ce6 Mon Sep 17 00:00:00 2001
From: Kent Fredric <kentnl@gentoo.org>
Date: Wed, 1 Jan 2020 21:35:17 +1300
Subject: Remove nightly-only features

packed-simd uses nightly only features, so this crate cannot be compiled
on stable with this feature.
---
 Cargo.toml | 6 +-----
 1 file changed, 1 insertion(+), 5 deletions(-)

diff --git a/Cargo.toml b/Cargo.toml
index 9497c8c..da1164e 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -27,10 +27,6 @@ lto = true
 [dependencies.cfg-if]
 version = "0.1.0"
 
-[dependencies.packed_simd]
-version = "0.3.3"
-optional = true
-
 [dependencies.serde]
 version = "1.0"
 optional = true
@@ -53,6 +49,6 @@ fast-legacy-encode = ["fast-hangul-encode", "fast-hanja-encode", "fast-kanji-enc
 less-slow-big5-hanzi-encode = []
 less-slow-gb-hanzi-encode = []
 less-slow-kanji-encode = []
-simd-accel = ["packed_simd", "packed_simd/into_bits"]
+
 [badges.travis-ci]
 repository = "hsivonen/encoding_rs"
-- 
2.24.0

